wishlist.toml

```diff
@@ -1105,6 +1105,12 @@ description = "DocuSign alternative, for signing and annotating PDF files"
 upstream = "https://github.com/OpenSignLabs/OpenSign"
 website = "https://www.opensignlabs.com/"
 
+[opentalk]
+name = "OpenTalk"
+description = "German developed, modern, conferencing tool running in browser alone."
```

Comment:
I prefer my software Earth-sourced. :/

Reply:
Well. I have access through my mail provider, and I can just say, that it is a solid piece of software. Id choose German over many other.

Reply:
Patriotism is not a relevant argument in the ynh catalog :”)